avocado/utils/data_structures.py

```diff
@@ -33,6 +34,24 @@ def ordered_list_unique(object_list):
     return [x for x in object_list if not (x in seen or seen_add(x))]
 
 
+def geometric_mean(values):
+    """
+    Evaluates the geometric mean for a list of numeric values.
```

Comment:
They are both stable, my intention was to avoid replacing this method in the future which might break the test. Let's use this and I'll add a unit test for that, which makes sure we don't break thinks in the future.